ihmc-communication/src/main/java/us/ihmc/communication/crdt/RequestConfirmFreezable.java

```diff
@@ -154,8 +161,6 @@ public void fromMessage(ConfirmableRequestMessage message)
             recentConfirmations.add(confirmationNumber);
             confirmationTimeouts.put(confirmationNumber, crdtInfo.getUpdateNumber() + crdtInfo.getMaxFreezeDuration());
          }
-
-         nextRequestID.setValue(confirmationNumber + 1); // Avoid duplicate request numbers for clarity
```

Comment:
This was for early debugging, removing it to make it simpler and not skip request numbers